dist/README.md

```diff
@@ -0,0 +1,3 @@
+# Downstream management
+
+Directory dedicated for various package managers to recognize this repository into a package
```

Comment:
WTF no please don't do something like this upstream. Package files should be downstream, always. It only bloats the repo and you can't even directly change something in there since you don't have write rights.